.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Release .deb files'
+on: release
+
```

Comment:
I'm not sure, but you may not have to check generated files (by template) to the repository by:
- runs generate_actions.py as a first job
- then fires all the releases jobs below.

Reply:
OK, perged `generate_actions.py`. I don't like doing acrobatic method on the github action.